portal/templates/profile_macros.html

```diff
@@ -836,9 +836,11 @@ <h4 class="text-muted">{{ _("Questions inquired of the patient at registration")
         };
         $(function () {
             $("#patientQ").show();
+            $("#patTx").remove();
```

Comment:
Not sure I totally grok the flow, but why are we removing #patTx here?  Shouldn't it just be removed from the profile macros source?